unity-renderer/Assets/Rendering/Outliner/OutlineScreenEffectFeature.cs

```diff
@@ -73,34 +90,36 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                 Blit(cmd, source, camera.id); // Get camera in a RT
                 cmd.SetGlobalTexture("_Source", camera.id); // Apply RT as _Source for the material
                 cmd.SetGlobalTexture("_ComposeMask", outlineMask.id); // Set the original outline mask
+
                 Blit(cmd, outline1.id, source, material, (int)ShaderPasses.Compose);
 
                 cmd.ReleaseTemporaryRT(camera.id);
                 cmd.ReleaseTemporaryRT(outline1.id);
                 cmd.ReleaseTemporaryRT(outline2.id);
-
-
             }
 
             context.ExecuteCommandBuffer(cmd);
             CommandBufferPool.Release(cmd);
         }
 
-        public override void FrameCleanup(CommandBuffer cmd)
-        {
-        }
+        private static bool IsOutlineAvailable() =>
+            DataStore.i.outliner.avatarOutlined.Get().renderer != null;
+
+        public override void FrameCleanup(CommandBuffer cmd) { }
     }
 
     [System.Serializable]
     public class OutlineSettings
     {
-        public float outlineSize = 0.001f;
-        public float blurSize = 0.001f;
-        public float blurSigma = 1;
         public Color outlineColor = Color.cyan;
         public Color blurColor = Color.cyan;
         public Color innerColor = Color.cyan;
         [Range(0, 1)] public float effectFade = 1;
+        public FloatVariable outlineThickness;
+        public FloatVariable outlineResolutionScale;
+        public FloatVariable blurSize;
+        public FloatVariable blurSigma;
```

Comment:
what is the benefit of having SO variables here instead of normal variables?

Reply:
Further, I left a comment that all these settings can be grouped in one ScriptableObject for OutlineSettings